import mysql.connector
# from mysql.connector import Error
ConnBD = mysql.connector.connect(host='172.17.0.2',
                              database='tester',
                              user='root',
                              password='7cronos1')


